arapp.json

```diff
@@ -16,6 +16,12 @@
       "network": "development",
       "appName": "delay.open.aragonpm.eth"
     },
+    "staging": {
+      "registry": "0x98df287b6c145399aaa709692c8d308357bc085d",
+      "appName": "delay-staging.open.aragonpm.eth",
+      "wsRPC": "wss://rinkeby.eth.aragon.network/ws",
+      "network": "rinkeby"
+    },
```

Comment:
This one is so that when publishing the app, we first do it in a staging repo before publishing to the main repo. The only thing is different is the appName.